### DISCUSSION TITLE
Dark Reader 4.9.51 - New & Improved! - Now with Infinity Color!

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#8966] [General]
I was just curious if there was a big change in Dark Reader 4.9.51 that would result in a MASSIVE improvement in how websites appear in general?

What I mean is that the black colors and backgrounds on websites are so much darker and other colors are so much more vibrant and really just pop. I don't know how to describe it exactly, but WOW! What a HUGE difference. It's like I went from having an old monitor or TV from 10-15 years ago and just got upgraded to a brand new OLED! (I know that might be seen as an exaggeration, but it really is strikingly different.)

There are several websites I visit everyday or almost everyday and I immediately noticed a difference. Some of them looked very drab and dreary with very muted colors with what previous versions of Dark Reader could do for them, however now they look very nice. The first few I visited after installing this new version, I legitimately did a double take and thought to myself that the website looked different or was more colorful.

All I can say is a BIG Thank You & Congratulations for taking Dark Reader up to the next level or two! :smiley:

**Comment by alexanderby** [comment]
Hi @q1800! Which browser and version were you using? Seems like you had some bug.

**Comment by ghost** [comment]
Nevermind... LOL! The contrast level in Dark Reader somehow got increased to 150%.
